pandas/tests/generic/test_series.py

```diff
@@ -134,3 +134,17 @@ def finalize(self, other, method=None, **kwargs):
         # reset
         Series._metadata = _metadata
         Series.__finalize__ = _finalize  # FIXME: use monkeypatch
+
+    def test_series_clipping_with_na_values(self):
```

Comment:
can you parameterize by nullable int & float types (I think we have a fixture for this)

Reply:
I parameterized the test but I could not find any clear listing of all nullable int and float types.